src/clickfunnels/main.ts

```diff
@@ -82,6 +103,25 @@ document.addEventListener('DOMContentLoaded', () => {
     return originalFetch.apply(this, args);
   };
 
+  // Attach listeners to all buttons currently on the page
+  document.querySelectorAll('.elButton').forEach((element: HTMLElement) => {
+    addButtonClickListener(element);
```

Comment:
Is there a case where the querySelectorAll can return null?

Reply:
not only an empty node list.